Learn-CSS-Blog/index36.html

```diff
@@ -0,0 +1,81 @@
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed imperdiet 
+                        fermentum tortor, at aliquet ante varius nec. Sed vehicula libero sed 
+                        ligula varius, sit amet eleifend velit tincidunt. Fusce eu justo quis 
+                        neque sagittis venenatis. Vivamus ac elit ultricies, laoreet leo vitae, 
+                        rutrum eros. Mauris malesuada auctor quam et facilisis. Nullam ipsum massa,
+                        ultricies id faucibus non, dapibus eu arcu. Fusce non interdum risus.
+                        Ut at lacus mattis, aliquam tellus at, luctus magna. Quisque euismod, 
+                        eros non dictum tincidunt, nisl mauris elementum nibh, vitae facilisis 
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed imperdiet 
+                        fermentum tortor, at aliquet ante varius nec. Sed vehicula libero sed 
+                        ligula varius, sit amet eleifend velit tincidunt. Fusce eu justo quis 
+                        neque sagittis venenatis. Vivamus ac elit ultricies, laoreet leo vitae, 
+                        rutrum eros. Mauris malesuada auctor quam et facilisis. Nullam ipsum massa,
+                        ultricies id faucibus non, dapibus eu arcu. Fusce non interdum risus.
+                        Ut at lacus mattis, aliquam tellus at, luctus magna. Quisque euismod, 
+                        eros non dictum tincidunt, nisl mauris elementum nibh, vitae facilisis 
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                </div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Add the `href` attribute with the value of `style.css` to the `<link>` tag.
```

Comment:
Add an `href` attribute with the value "style.css" to the `<link>` element.
    The `href` attribute specifies the location (URL) of the external resource,
    in this case, the `style.css` file.